.bazelrc

```diff
@@ -12,38 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+###############################################################################
+# Common flags that apply to all configurations.
+# Use sparingly because they cannot be overriden.
```

Comment:
This shouldn't be true in general. We set `--compilation_mode=opt`, but that doesn't prevent `--compilation_mode=fastbuild` from working.

I guess the issue is that *repeated* flags can't be removed, only overridden, which doesn't work for a copt that the compiler totally doesn't know about.

Reply:
Rephrased.